internal/cli/actions.go

```diff
@@ -262,15 +274,15 @@ func updateActionCmd(cli *cli) *cobra.Command {
 		Short: "Update an action",
 		Long: "Update an action.\n\n" +
 			"To update interactively, use `auth0 actions update` with no arguments.\n\n" +
-			"To update non-interactively, supply the action id, name, trigger, secrets and " +
+			"To update non-interactively, supply the action id, name, cod, secrets and " +
 			"dependencies through the flags.",
 		Example: `  auth0 actions update <action-id> 
   auth0 actions update <action-id> --name myaction
-  auth0 actions update <action-id> --name myaction --trigger post-login
-  auth0 actions update <action-id> --name myaction --trigger post-login --code "$(cat path/to/code.js)"
-  auth0 actions update <action-id> --name myaction --trigger post-login --code "$(cat path/to/code.js)" --dependency "lodash=4.0.0"
-  auth0 actions update <action-id> --name myaction --trigger post-login --code "$(cat path/to/code.js)" --dependency "lodash=4.0.0" --secret "SECRET=value"
-  auth0 actions update <action-id> --name myaction --trigger post-login --code "$(cat path/to/code.js)" --dependency "lodash=4.0.0" --dependency "uuid=9.0.0" --secret "API_KEY=value" --secret "SECRET=value"
+  auth0 actions update <action-id> --name myaction
```

Comment:
```suggestion
```
This one is identical to the one above.

Reply:
Fixed in [`c3e12bd` (#597)](https://github.com/auth0/auth0-cli/pull/597/commits/c3e12bd7a9c95d4dfc699c375ad5d9ddaa9a2fb4)